### PR TITLE
[TIPC] update  tipc

### PR DESCRIPTION
**Pull request by andyjpaddle**
update train infer python for tipc
add cpp infer for tipc
add paddle2onnx for tipc

### CODE REVIEW

File: test_tipc/configs/ch_PP-OCRv2/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+use_opencv:True
+infer_model:./inference/ch_PP-OCRv2_det_infer/
+infer_quant:False
+inference:./deploy/cpp_infer/build/ppocr --rec_char_dict_path=./ppocr/utils/ppocr_keys_v1.txt
+--use_gpu:True|False
+--enable_mkldnn:False
+--cpu_threads:6
+--rec_batch_num:1
+--use_tensorrt:False
+--precision:fp32|fp16
```
Comment (LDOUBLEV) [inline test_tipc/configs/ch_PP-OCRv2/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt:12]: precision设置一个fp32就够了，不开启TRT、mkldnn，precisio默认fp32

File: test_tipc/configs/ch_PP-OCRv2_det/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+use_opencv:True
+infer_model:./inference/ch_PP-OCRv2_det_infer/
+infer_quant:False
+inference:./deploy/cpp_infer/build/ppocr
+--use_gpu:True|False
+--enable_mkldnn:False
+--cpu_threads:6
+--rec_batch_num:1
+--use_tensorrt:False
+--precision:fp32|fp16
```
Comment (LDOUBLEV) [inline test_tipc/configs/ch_PP-OCRv2_det/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt:12]: 同上

File: test_tipc/test_paddle2onnx.sh [hunk 3]
```diff
@@ -49,34 +54,91 @@ function func_paddle2onnx(){
...
+        set_dirname=$(func_set_params "--model_dir" "${det_infer_model_dir_value}")
+        set_model_filename=$(func_set_params "${model_filename_key}" "${model_filename_value}")
+        set_params_filename=$(func_set_params "${params_filename_key}" "${params_filename_value}")
+        set_save_model=$(func_set_params "--save_file" "${det_save_file_value}")
+        set_opset_version=$(func_set_params "${opset_version_key}" "${opset_version_value}")
+        set_enable_onnx_checker=$(func_set_params "${enable_onnx_checker_key}" "${enable_onnx_checker_value}")
+        trans_model_cmd="${padlle2onnx_cmd} ${set_dirname} ${set_model_filename} ${set_params_filename} ${set_save_model} ${set_opset_version} ${set_enable_onnx_checker}"
+        eval $trans_model_cmd
+        last_status=${PIPESTATUS[0]}
+        status_check $last_status "${trans_model_cmd}" "${status_log}"
```
Comment (LDOUBLEV) [inline test_tipc/test_paddle2onnx.sh:68]: status_check的函数加上model_name参数，参考这个：https://github.com/PaddlePaddle/models/blob/b7e312d2c451cff73376551ecbe141ccbdbf821c/tutorials/mobilenetv3_prod/Step6/test_tipc/test_train_inference_python.sh#L90
新的statuscheck实现：https://github.com/PaddlePaddle/models/blob/b7e312d2c451cff73376551ecbe141ccbdbf821c/tutorials/mobilenetv3_prod/Step6/test_tipc/common_func.sh#L64

Comment (andyjpaddle) [reply test_tipc/test_paddle2onnx.sh]: done,ths

File: test_tipc/configs/ch_PP-OCRv2/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt (hunk 1 above)
Comment (andyjpaddle) [reply test_tipc/configs/ch_PP-OCRv2/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt]: done,ths

File: test_tipc/configs/ch_PP-OCRv2_det/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt (hunk 2 above)
Comment (andyjpaddle) [reply test_tipc/configs/ch_PP-OCRv2_det/model_linux_gpu_normal_normal_infer_cpp_linux_gpu_cpu.txt]: done,ths

**Review by LDOUBLEV — APPROVED** [review]

**Pull request merged**